tests/README.md

```diff
@@ -38,17 +38,17 @@ correctly, or to extract the correct name to run a one shot test.
 To manually run one shot tests, first [connect to a cluster](https://console.cloud.google.com/kubernetes/list) and then run the following:
 
 ```bash
-export TEST_NAME=tf.nightly-vit-beans-func-v4-8-1vm
+export TEST_NAME=tf.nightly-dlrm-criteo-conv-v100-x1
 jsonnet tests/oneshot.jsonnet -J . -S --tla-str test=$TEST_NAME | kubectl create -f -
 ```
 
-The above command will generate a job id such as `job.batch/pt-nightly-unet3d-conv-v3-8-1vm-gz8ww`. To find the detail of the test, search in [GoogleCloud->Kubernetes->workload in the project `xl-ml-test`](https://console.cloud.google.com/kubernetes/workload/overview?mods=allow_workbench_image_override&project=xl-ml-test) with the job id `pt-nightly-unet3d-conv-v3-8-1vm-gz8ww`.
+The above command will generate a job id such as `job.batch/tf.nightly-dlrm-criteo-conv-v100-x1`. To find the detail of the test, search in [GoogleCloud->Kubernetes->workload in the project `xl-ml-test`](https://console.cloud.google.com/kubernetes/workload/overview?mods=allow_workbench_image_override&project=xl-ml-test) with the job id `tf.nightly-dlrm-criteo-conv-v100-x1`.
```

Comment:
small nit: in the current example we see `job.batch/pt-nightly-unet3d-conv-v3-8-1vm-gz8ww` (namely the gz8ww, which isn't part of the test name). The `gz8ww` is there to uniquely identify different one-shots. I suggest we leave that in. While technically there is not an actual one-shot with the name `job.batch/tf.nightly-dlrm-criteo-conv-v100-x1-gz8ww`, it is fine as an example of what a job id might look like.

Reply:
Whoops, thought I left that in